apps/meteor/app/livechat/server/api/v1/config.ts

```diff
@@ -23,12 +24,12 @@ API.v1.addRoute(
 			const config = await cachedSettings({ businessUnit });
 
 			const status = Livechat.online(department);
-			const guest = token && (await Livechat.findGuest(token));
+			const guest: ILivechatVisitor | null = token ? await Livechat.findGuest(token) : null;
```

Comment:
declaring the type for `guest` is not wrong, but since TS can infer it correctly, removing it improves code readability.. wdyt?
```suggestion
			const guest = token ? await Livechat.findGuest(token) : null;
```